domain/src/test/java/org/oppia/android/domain/classify/rules/fractioninput/FractionInputHasDenominatorEqualToRuleClassifierProviderTest.kt

```diff
@@ -25,11 +26,17 @@ import kotlin.test.fail
 @LooperMode(LooperMode.Mode.PAUSED)
 @Config(manifest = Config.NONE)
 class FractionInputHasDenominatorEqualToRuleClassifierProviderTest {
-  private val WHOLE_NUMBER_123 = createWholeNumber(isNegative = false, value = 123)
-  private val FRACTION_1_OVER_2 = createFraction(isNegative = false, numerator = 1, denominator = 2)
-  private val FRACTION_2_OVER_4 = createFraction(isNegative = false, numerator = 2, denominator = 4)
-  private val NON_NEGATIVE_VALUE_1 = createNonNegativeInt(value = 1)
-  private val NON_NEGATIVE_VALUE_2 = createNonNegativeInt(value = 2)
+
+  private val WHOLE_NUMBER_123 =
+    createWholeNumber(isNegative = false, value = 123)
```

Comment:
For the consistency let's access methods using the object, not importing directly. 
`InteractionObjectTestBuilder`. 

@aggarwalpulkit596 any suggestion on this? Either we should import directly or using object, which could be a good practice?

Reply:
if we write `InteractionObjectTestBuilser.createMixedNumber(isNegative = false, wholeNumber = 123, numerator = 1, denominator = 2)` then the length of this line goes over 100 words. Should I remove `isNegative` and other such values ?

Reply:
the line can be broken down easily, but keep the consistency. 

```kotlin
= InteractionObjectTestBuilser.createMixedNumber(
    isNegative = false,
    wholeNumber = 123,
    numerator = 1,
    denominator = 2
  )
```

Reply:
done